examples/example.rst

```diff
@@ -0,0 +1,23 @@
+Examples
+========
+
+This are example application of using `Flask-Reuploaded`. 
```

Comment:
```suggestion
These are example applications of using `Flask-Reuploaded`. 
```

Reply:
For `README.rst`, I will restore the previous version without applying any of the suggestions. Just for this pull request. For the next pull request I'll try to apply all suggestions.

[restore README](https://github.com/jugmac00/flask-reuploaded/commit/b2fe9638107b9059081d5ffc3ba25f163160fe20)